core/src/main/java/bisq/core/xmr/connection/MoneroConnectionsManager.java

```diff
@@ -0,0 +1,209 @@
+package bisq.core.xmr.connection;
+
+import bisq.core.api.model.UriConnection;
```

Comment:
The MoneroConnectionsManager class should only use MoneroRpcConnections. UriConnection is only for the grpc api, so any reference to it should be moved to CoreMoneroConnectionsService, which can handle translation between UriConnection and MoneroRpcConnection.